src/main/java/org/jahia/modules/newsletter/action/UnsubscribeAction.java

```diff
@@ -130,6 +135,32 @@ public static String generateUnsubscribeLink(JCRNodeWrapper newsletterNode, Stri
 		        + newsletterNode.getLanguage() + newsletterNode.getPath()
 		        + ".confirm.do?key=" + confirmationKey + "&exec=rem";
 	}
+	
+	/**
+	* Check if templatSet has mail template
+	*/
+    private String getTemplateName(String template, JCRNodeWrapper node,  final Locale locale, String defaultTemplate){
+    	String templateToReturn = defaultTemplate;
+
+    	try {
+	    	//try if it is multilingual
```

Comment:
This code is duplicate from SubscribeAction.
We need to centralize this code in one unique place.